src/utilities/localStorage.js

```diff
@@ -0,0 +1,35 @@
+export function getDataFromLocalStorage() {
+  const data = JSON.parse(localStorage.getItem("restaurant")) || [];
+
+  return data;
+}
+
+export function sendDataToLocalStorage(items) {
+  const data = getDataFromLocalStorage();
+
+  data.push(items);
+  localStorage.setItem("restaurant", JSON.stringify(data));
+}
+
+export function removeDataFromLocalStorageById(restaurantId) {
+  const data = getDataFromLocalStorage();
+
+  const newData = data.filter((item) => {
+    return item.restaurantId !== restaurantId;
+  });
+
+  localStorage.setItem("restaurant", JSON.stringify(newData));
+}
+
+export function getNameFromLocalStorage() {
+  const data = JSON.parse(localStorage.getItem("name")) || [];
```

Comment:
Why not an empty string instead of an empty array. I am assuming you want to save a username in local storage, this really should be a string not an array, also regarding your `sendNameToLocalStorage` function